imports/client/components/NotificationCenter.tsx

```diff
@@ -81,28 +82,26 @@ const StyledNotificationMessage = styled.li`
 `;
 
 const StyledNotificationActionBar = styled.ul`
-  display: block;
-  list-style-type: none;
-  margin: 0px;
-  padding: 0px;
   display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
   flex-direction: row;
 `;
 
 const StyledNotificationActionItem = styled.li`
-  margin-left: 0px;
-  margin-top: 8px;
-  margin-right: 8px;
-  margin-bottom: 4px;
+  margin: 8px 8px 4px 0;
```

Comment:
In all my years of webdev I have never remembered the order of these ;_; I can probably be taught but I admit I find this less readable than how it was

Reply:
It's clockwise, if that helps